### PR TITLE
Added a question about big O for linear time

### PR DESCRIPTION
**Pull request by sotoblanco**
This is a fundamental question to ensure the student understands linear running time big O notation.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1430,7 +1430,17 @@ const generalCSQuiz = [
...
     Link: "https://en.wikipedia.org/wiki/Fizz_buzz"
-  }
+  },
+  {
+    Question: "What's the Big O notation of an algorithm that increases the running time linearly with the input size?" ,
+    Answer: "O(n)" ,
+    Distractor1: "O(n^2)" ,
+    Distractor2: "O(1)" ,
+    Distractor3: "O(log n)" ,
+    Explanation: "The letter n represents the input size, the big O represents the increase of the running time." ,
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1440]:
```suggestion
    Explanation: "The letter n represents the input size, and the big O represents the increase of the running time." ,
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by sotoblanco** [comment]
Thank you!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**